laravel/database/migrations/2020_09_06_154452_create_items_table.php

```diff
@@ -15,9 +15,9 @@ public function up()
     {
         Schema::create('items', function (Blueprint $table) {
             $table->bigIncrements('id');
-            $table->string('item_code', 11);
+            $table->string('item_code', 11)->index()->unique();
```

Comment:
ユニークなども意識できていて良いですね 👍 
ただユニークを貼ると自動でインデックスもはってくれるので、indexの記述は不要だった気がします
```suggestion
            $table->string('item_code', 11)->unique();
```